techniques/applications/systemUpdateCampaign/1.0/metadata.xml

```diff
@@ -0,0 +1,71 @@
+<TECHNIQUE name="System Update Campaign">
+  <DESCRIPTION>This technique can handle simple system wide updates on different Linux based systems.
+
+### Configuration
+
+Each update is unique and *must* be tagged by a unique `campaign name`. The name will be used to reference the scheduled system update by the agent.
+Updates can be scheduled by giving a scheduling expression and a duration. The update is only run once, and there is no retry in case of error.
+The error will not persist so you will have to check manually in the node report it was successful or not. Except at update time, every report generated
+by the technique will have the `not_applicable` status so it should be easy to identify the proper execution time in the logs.
+
+### Post update
+
+The technique will automatically run a Rudder inventory after the update if it was successful. A system reboot can also be triggered if needed.
+</DESCRIPTION>
+  <MULTIINSTANCE>false</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
+  <USEMETHODREPORTING>true</USEMETHODREPORTING>
```

Comment:
if it's a system technique, you should add <SYSTEM>true</SYSTEM

Reply:
how will I be able to create a directive based on it if it is system?

Reply:
isn't it created with the plugin ?